### PR TITLE
Firebase managing admins and toggle form

### PR DESCRIPTION
**Pull request by kerrd89**

### CODE REVIEW

**Comment by kerrd89** [comment]
closes #28

File: lib/components/App.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ export default class App extends Component {
     this.state = {
       user: null,
       spikes: [],
+      admins: [],
```
Comment (swanie21) [inline lib/components/App.js]: Yes, we needed an admin list 👍

File: lib/components/Admin.js [hunk 2]
```diff
@@ -24,10 +24,12 @@ const Admin = ({ spikes, createSpike, updateSpike, deleteSpike, user, admins })
 
   return (
     <div className="Admin">
-      <Spike
+      {!showForm && <button onClick={() => toggleForm()}>Add Spike</button>}
```
Comment (swanie21) [inline lib/components/Admin.js]: This really cleans up the pages.

**Comment by swanie21** [comment]
Wow, you sure cleaned up some files, looking good 😁

**Pull request merged**